pkgs/development/libraries/science/math/mkl/default.nix

```diff
@@ -139,6 +151,11 @@ in stdenvNoCC.mkDerivation {
     install_name_tool -change @rpath/libtbbmalloc.dylib $out/lib/libtbbmalloc.dylib $out/lib/libtbbmalloc_proxy.dylib
   '';
 
+  # Validate pkgconfig files, since they break often on updates.
```

Comment:
It might be nice to do this as a generic stdenv hook fix (opt-in).  pcfiles seem to break often, e.g. just in the last couple weeks:
https://github.com/NixOS/nixpkgs/pull/85254

That said, this is already a nice improvement!

Reply:
That's a great idea! I already looked whether there is such a hook before submitting this PR. I'll give it a try tomorrow or coming weekend.

Reply:
Sounds good; in the meantime this is already a material improvement, so I'm going to merge this and we can do the larger fix in a follow up. Can you send a 20.03 backport PR, too?

Reply:
Will do!